docs/user_workloads/accessing_virtual_machines.md

```diff
@@ -327,29 +334,32 @@ and traffic consider using a dedicated Kubernetes `Service` instead.
 2. SSH into virtual machine
 
 ```shell
-# Add --local-ssh to transparently use local OpenSSH client
-virtctl ssh -i id_rsa fedora@testvm
+virtctl ssh -i id_rsa fedora@vm/testvm/mynamespace
 ```
 
 or
 
 ```shell
-ssh -o 'ProxyCommand=virtctl port-forward --stdio=true vmi/testvm.mynamespace 22' -i id_rsa fedora@vmi/testvm.mynamespace
+ssh -o 'ProxyCommand=virtctl port-forward --stdio=true vm/testvm/mynamespace 22' -i id_rsa fedora@vm/testvm/mynamespace
 ```
 
-4. SCP file to the virtual machine
+3. SCP file to the virtual machine
 
 ```shell
-# Add --local-ssh to transparently use local OpenSSH client
-virtctl scp -i id_rsa testfile fedora@testvm:/tmp
+virtctl scp -i id_rsa testfile fedora@vm/testvm/mynamespace:/tmp
 ```
 
 or
 
 ```shell
-scp -o 'ProxyCommand=virtctl port-forward --stdio=true vmi/testvm.mynamespace 22' -i id_rsa testfile fedora@testvm.mynamespace:/tmp
+scp -o 'ProxyCommand=virtctl port-forward --stdio=true vmi/testvm/mynamespace 22' -i id_rsa testfile fedora@vm.testvm.mynamespace:/tmp
 ```
 
+!!! Note
+    Local `scp` does not support slashes in hostnames. Therefore the example
+    uses dots to separate the type, name and namespace like `virtctl` does when
+    wrapping the local SCP client.
```

Comment:
Nit: this shows as-is and is not interpreted as a markdown Note block, maybe we want to make it one?

Reply:
It looks fine in the preview though. Maybe it is an extension of the page generator we are using?

Reply:
Yes, it's an admonition format for material so it will render nicely in our built docs/netlify preview but not in the github preview.